payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/events/CardNumberCompletedEventReporter.kt

```diff
@@ -0,0 +1,23 @@
+package com.stripe.android.ui.core.elements.events
+
+import androidx.annotation.RestrictTo
+import androidx.compose.runtime.staticCompositionLocalOf
+import com.stripe.android.uicore.BuildConfig
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun interface CardNumberCompletedEventReporter {
+    fun onCardNumberCompleted(): Unit
+}
```

Comment:
We also have an autofill event reporter. We should consolidate them into a single event reporter.